src/main/java/org/terasology/tutorialWorldGeneration/TutorialWorldRasterizer.java

```diff
@@ -42,12 +42,12 @@ public void initialize() {
     public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         ElevationFacet elevationFacet = chunkRegion.getFacet(ElevationFacet.class);
         SurfacesFacet surfacesFacet = chunkRegion.getFacet(SurfacesFacet.class);
-        for (Vector3i position : chunkRegion.getRegion()) {
-            float surfaceHeight = elevationFacet.getWorld(position.x, position.z);
-            if (surfacesFacet.getWorld(JomlUtil.from(position))) {
-                chunk.setBlock(ChunkMath.calcRelativeBlockPos(position), grass);
-            } else if (position.y < surfaceHeight) {
-                chunk.setBlock(ChunkMath.calcRelativeBlockPos(position), dirt);
+        for (Vector3ic position : chunkRegion.getRegion()) {
+            float surfaceHeight = elevationFacet.getWorld(position.x(), position.z());
+            if (surfacesFacet.getWorld(position)){
+                chunk.setBlock(Chunks.toRelative((org.joml.Vector3i) position), grass);
+            } else if (position.y() < surfaceHeight) {
+                chunk.setBlock(Chunks.toRelative((org.joml.Vector3i) position), dirt);
```

Comment:
Take care about such inlined imports.
Make sure to import the respective class properly at the top of the file.